extensions/aproc/proc/ingest/drivers/impl/spot5.py

```diff
@@ -117,19 +122,21 @@ def to_item(self, url: str, assets: list[Asset]) -> Item:
             ),
             assets=dict(map(lambda asset: (asset.name, asset), assets))
         )
+
+        if dim_path != self.dim_path:
+            os.remove(dim_path)
```

Comment:
The system should never delete any file. If it is really needed, then this must be documented and we must use the comment "!DELETE!".

https://docs.google.com/document/d/1UTm5JUZ8F9ptlcya9v88hzg-iPWxeuCCziAD_oycloo/edit?tab=t.0